### DISCUSSION TITLE
OpenSSL 3.0 Fips provider  question on install-mac, module-mac field

### DISCUSSION THREAD

**Discussion by aniljadaun** [openssl/openssl#21750] [Q&A] [answered]
Hi

The fipsmodule.cnf file created through the execution of the fipsinstall command lacks the install-mac parameter, yet it contains the module-mac parameter. The FIPS provider has been successfully loaded as intended.

My inquiry pertains to whether it is obligatory for the fipsmodule.cnf file to encompass both the module-mac and install-mac parameters, or if solely having the module-mac parameter is satisfactory for achieving FIPS compliance.

**Comment by t8m** [comment, marked as answer]
Missing install-mac triggers always running the power-on self tests. That should still keep the module FIPS compliant in the sense of technical FIPS requirements. Not running fipsinstall is though in breach of the security policy of the module which means not compliant by letter.

You should contact a FIPS validation lab to get authoritative answers to questions related to FIPS compliance though.

**Reply by aniljadaun** [reply]
Thanks
